pages/services/hdfs/2.5.0-2.6.0-cdh5.11.0/release-notes/index.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+excerpt: Release notes for v. 2.5.0-2.6.0-cdh5.11.0
+title: Release Notes
+menuWeight: 10
+model: /services/hdfs/data.yml
+render: mustache
+---
+## Version 2.5.0-2.6.0-cdh5.11.0
+### Feature
```

Comment:
all the three features below describe some parameter exposed as a configurable property but do it in a different way:
- "... now available in service configuration"
- "... now available as configurable property"
- "... now a configurable property"

Shouldn't the phrasing be the same if we're are speaking about the same things pretty much?

Reply:
@soumasish how about this comment?

Reply:
I made the changes, modified the text to be consistent.